.env

```diff
@@ -254,7 +254,7 @@ DREVOPS_COMMIT=
 #;< !DATABASE_DOWNLOAD_SOURCE_DOCKER_REGISTRY
 # Location of the demo database used for demonstration with CURL database
 # dump as a type of file source.
-CURL_DB_URL=https://raw.githubusercontent.com/wiki/drevops/drevops/db_d8.dist.sql.md
+CURL_DB_URL=https://raw.githubusercontent.com/wiki/drevops/drevops/db_d9.dist.sql.md
```

Comment:
this and below - would need to base the file name on `$DRUPAL_VERSION` so that these changes are no longer required

Reply:
Is this PR the place to do it? Or are you noting for future PR / improvement.